### PR TITLE
[s]Fixes fake nukes ending the round

### PR DESCRIPTION
**Pull request by Cyberboss**

### CODE REVIEW

**Comment by MrStonedOne** [comment]
Not really a secret as only admins can do these right?

right?

**Comment by Cyberboss** [comment]
Wizard

File: code/controllers/subsystem/ticker.dm [hunk 1]
```diff
@@ -331,26 +326,27 @@ var/datum/subsystem/ticker/ticker
...
-		mode.station_was_nuked = (station_missed<2)	//station_missed==1 is a draw. the station becomes irradiated and needs to be evacuated.
-														//kinda shit but I couldn't  get permission to do what I wanted to do.
 	var/bombloc = null
 	if(actually_blew_up)
 		if(bomb && bomb.loc)
 			bombloc = bomb.z
 		else if(!station_missed)
 			bombloc = ZLEVEL_STATION
+		
+		if(mode)mode)
```
Comment (francinum) [inline code/controllers/subsystem/ticker.dm]: This doesn't look right.

Comment (kevinz000) [inline code/controllers/subsystem/ticker.dm]: :^)

Comment (Cyberboss) [inline code/controllers/subsystem/ticker.dm]: 12:47 am

**Comment by SweaterKittens** [comment]
I'm assuming this was brought on by an admin accidentally ending the round with this while attempting to play it as a joke.

**Comment by AnturK** [comment]
Check if it works with no-core nuke too, it had some unique call iirc.

**Comment by Cyberboss** [comment]
No core def works because it fast returns

**Comment by Incoming5643** [comment]
It's a weight 0 wizard event that's still in the codebase just for admin use, summon events can't draw it any longer.

**Pull request merged**